test_main.c

```diff
@@ -17,13 +17,20 @@
 
 #include "tests.h"
 
+int runverbose = 1;
+
+const char* vers      = "\\0$VER: ast_tests 1.0.0";
+const char* min_stack = "$STACK:16384";
+
 int main(int argc, const char *argv[]) {
   struct TestResult result;
 
   result.ran = 0;
   result.passed = 0;
   result.failed = 0;
   
+  if( argc > 1 ) runverbose = 0;
```

Comment:
if we pass *anything* as an arg, switch to quiet mode.  I could have done a strcmp but YAGNI.